providers/common/der/build.info

```diff
@@ -25,43 +25,49 @@ GENERATE[$DER_RSA_H]=der_rsa.h.in
 DEPEND[$DER_RSA_H]=oids_to_c.pm
 
 #----- DSA
-$DER_DSA_H=../include/prov/der_dsa.h
-$DER_DSA_GEN=der_dsa_gen.c
-$DER_DSA_AUX=der_dsa_key.c der_dsa_sig.c
-
-GENERATE[$DER_DSA_GEN]=der_dsa_gen.c.in
-DEPEND[$DER_DSA_GEN]=oids_to_c.pm
-
-DEPEND[${DER_DSA_AUX/.c/.o}]=$DER_DSA_H $DER_DIGESTS_H
-DEPEND[${DER_DSA_GEN/.c/.o}]=$DER_DSA_H
-GENERATE[$DER_DSA_H]=der_dsa.h.in
-DEPEND[$DER_DSA_H]=oids_to_c.pm
+IF[{- !$disabled{dsa} -}]
+  $DER_DSA_H=../include/prov/der_dsa.h
+  $DER_DSA_GEN=der_dsa_gen.c
+  $DER_DSA_AUX=der_dsa_key.c der_dsa_sig.c
+
+  GENERATE[$DER_DSA_GEN]=der_dsa_gen.c.in
+  DEPEND[$DER_DSA_GEN]=oids_to_c.pm
+
+  DEPEND[${DER_DSA_AUX/.c/.o}]=$DER_DSA_H $DER_DIGESTS_H
+  DEPEND[${DER_DSA_GEN/.c/.o}]=$DER_DSA_H
+  GENERATE[$DER_DSA_H]=der_dsa.h.in
+  DEPEND[$DER_DSA_H]=oids_to_c.pm
+ENDIF
 
 #----- EC
-$DER_EC_H=../include/prov/der_ec.h
-$DER_EC_GEN=der_ec_gen.c
-$DER_EC_AUX=der_ec_key.c der_ec_sig.c
+IF[{- !$disabled{ec} -}]
+  $DER_EC_H=../include/prov/der_ec.h
+  $DER_EC_GEN=der_ec_gen.c
+  $DER_EC_AUX=der_ec_key.c der_ec_sig.c
 
-GENERATE[$DER_EC_GEN]=der_ec_gen.c.in
-DEPEND[$DER_EC_GEN]=oids_to_c.pm
+  GENERATE[$DER_EC_GEN]=der_ec_gen.c.in
+  DEPEND[$DER_EC_GEN]=oids_to_c.pm
 
-DEPEND[${DER_EC_AUX/.c/.o}]=$DER_EC_H $DER_DIGESTS_H
-DEPEND[${DER_EC_GEN/.c/.o}]=$DER_EC_H
-GENERATE[$DER_EC_H]=der_ec.h.in
-DEPEND[$DER_EC_H]=oids_to_c.pm
+  DEPEND[${DER_EC_AUX/.c/.o}]=$DER_EC_H $DER_DIGESTS_H
+  DEPEND[${DER_EC_GEN/.c/.o}]=$DER_EC_H
+  GENERATE[$DER_EC_H]=der_ec.h.in
+  DEPEND[$DER_EC_H]=oids_to_c.pm
+ENDIF
 
 #----- ECX
-$DER_ECX_H=../include/prov/der_ecx.h
-$DER_ECX_GEN=der_ecx_gen.c
-$DER_ECX_AUX=der_ecx_key.c
+IF[{- !$disabled{ec} -}]
```

Comment:
Couldn't we just merge this into the immediately prior "no-ec" block?

Reply:
I also thought about this but the current way is slightly nicer aligned with the structure/comments given in the file.

Reply:
What @DDvO said

Reply:
ok